test/proper_tests.erl

```diff
@@ -982,7 +983,7 @@ false_props_test_() ->
      ?_fails(error_statem:prop_simple())].
 
 error_props_test_() ->
-    [?_errorsOut({cant_generate,_},
+    [?_errorsOut({cant_generate,{proper_tests,error_props_test_,0}},
```

Comment:
`?MODULE` should be better here instead of `proper_tests`.
Same for line 1036 below.